docs/volunteers/education/google-classroom.md

```diff
@@ -3,6 +3,30 @@ id: google-classroom
 title: Google Classroom
 ---
 
-//TODO
+### Joining Google Clasroom
+
+In most cases, one of the mentors, or perhaps the course co-ordinator, will have access to the Classroom. They may even be the person who set it up. For you to have access, they will need to invite you via email. Once you receive the email, simply follow the invite's instructions.
+
```

Comment:
double space here `the  Classroom` and also maybe `in` over `on`, but I'm not a guru of prepositions 😛